src/main/java/com/google/step/YTLounge/data/Video.java

```diff
@@ -13,4 +13,20 @@ public Video(String title, String thumbnailURL, String videoURL, long duration)
     this.videoURL = videoURL;
     this.duration = duration;
   }
+
+  public String getTitle() {
+    return title;
+  }
+
+  public String getThumbnail() {
```

Comment:
do you want to call it getThumbnailURL for consistency?

Reply:
Sure, I'll change that